django/core/files/uploadhandler.py

```diff
@@ -127,6 +127,13 @@ def upload_complete(self):
         """
         pass
 
+    def upload_stopped(self):
+        """
+        Signal that the upload has been interrupted. Subclasses should perform cleanup that
+        is necessary for this handler.
```

Comment:
Can you wrap at 79 chars? as suggested in https://github.com/django/django/pull/13350#discussion_r481046575